agent/consul/acl.go

```diff
@@ -559,25 +460,33 @@ func (r *ACLResolver) resolvePoliciesAsyncForIdentity(identity structs.ACLIdenti
 		// again
```

Comment:
Might want to update the comment above since the result firing functionality has been removed.